Developer-Guide.md

```diff
@@ -277,10 +280,10 @@ $ tar -xf ${kernel_tar_file}
 $ mv .config "linux-${kernel_version}"
 $ pushd "linux-${kernel_version}"
 $ curl -L https://raw.githubusercontent.com/kata-containers/packaging/master/kernel/patches/0001-NO-UPSTREAM-9P-always-use-cached-inode-to-fill-in-v9.patch | patch -p1
-$ make ARCH=${kernel_arch} -j$(nproc)
+$ make ARCH=${kernel_dir} -j$(nproc)
 $ kata_kernel_dir="/usr/share/kata-containers"
 $ kata_vmlinuz="${kata_kernel_dir}/kata-vmlinuz-${kernel_version}.container"
-$ sudo install -o root -g root -m 0755 -D "$(realpath arch/${kernel_arch}/boot/bzImage)" "${kata_vmlinuz}"
+$ if [ $kernel_arch == ppc64le ]; then sudo install -o root -g root -m 0755 -D "$(realpath ./vmlinux)" "${kata_vmlinuz}"; else sudo install -o root -g root -m 0755 -D "$(realpath arch/${kernel_arch}/boot/bzImage)" "${kata_vmlinuz}"; fi
```

Comment:
I'd be tempted to simplify this further by removing the `if` and only defining the `install` line once. Something like:

```
$ ["$kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" || kernel_file="$(realpath arch/${kernel_arch}/boot/bzImage)"
$ sudo install -o root -g root -m 0755 -D "$kernel_file" "${kata_vmlinuz}"
```